src/donejs-carousel-plugin-test.js

```diff
@@ -1026,4 +1030,130 @@ describe('tucows-donejs-carousel', () => {
 	describe('clearAutoPlay()', () => {
 		// NOTE: We are not currently testing DOM manipulation
 	});
+
+	describe('Demo Page', () => {
```

Comment:
@tracer99 @ranjanbuilds am i on the right track with the two tests I added in this file? Vlad is currently preparing a list of things to check for, but I figured it wouldn't hurt to get a head start just ensuring that each of the `carouselOptions` does what it should do.

Reply:
Looks good.